index.html

```diff
@@ -405,18 +409,20 @@
     </section>
     <section>
       <h2>The `timeRemaining` method</h2>
-      <p>When the `timeRemaining()` method is invoked on an `IdleDeadline`
-      object it MUST return the duration, as a DOMHighResTimeStamp, between the
-      current time and the `deadline` argument passed to <a>IdleDeadline</a>.
+      <p>When the `timeRemaining()` method is invoked on an <a>IdleDeadline</a>
+      object it MUST return the duration, as a [DOMHighResTimeStamp], between
+      the current time and the [internal slot] on <a>IdleDeadline</a> which
+      stores it's absolute deadline.
       The value SHOULD be accurate to 5 microseconds - see "Privacy and
       Security" section of [[!HR-TIME]]. This value is calculated by performing
       the following steps:</p>
       <ol>
-        <li>Let <var>now</var> be the value returned by
-        `[performance.now()]`.</li>
-        <li>Let <var>timeRemaining</var> be `deadline` argument passed to
-        <a>IdleDeadline</a> - <var>now</var>.
-        </li>
+        <li>Let <var>now</var> be a [DOMHighResTimeStamp] representing
+        the current time in milliseconds.</li>
```

Comment:
Should links this to https://w3c.github.io/hr-time/#dfn-current-high-resolution-time